`test_mysql_protocol` is flaky

Integration test `test_mysql_protocol` fails with "image not found" from time to time:

[Test case history](https://play.clickhouse.com/play?user=play#U0VMRUNUIGJhc2VfcmVwbywgcHVsbF9yZXF1ZXN0X251bWJlciBhcyBwciwgY2hlY2tfc3RhcnRfdGltZSwgY2hlY2tfbmFtZSwgdGVzdF9uYW1lLCB0ZXN0X3N0YXR1cywgcmVwb3J0X3VybC0tLCB0ZXN0X2NvbnRleHRfcmF3CkZST00gY2hlY2tzCldIRVJFIDEKICAgIC0tQU5EIHB1bGxfcmVxdWVzdF9udW1iZXIgPSAwCiAgICBBTkQgdGVzdF9zdGF0dXMgSU4gKCdGQUlMJywgJ0VSUk9SJywgJ0JST0tFTicpCiAgICBBTkQgdG9EYXRlKGNoZWNrX3N0YXJ0X3RpbWUpID4gJzIwMjUtMDYtMDEnCiAgICBBTkQgdGVzdF9uYW1lIGxpa2UgJyV0ZXN0X215c3FsX3Byb3RvY29sJScKICAgIC0tIEFORCB0ZXN0X2NvbnRleHRfcmF3IGxpa2UgJyVBTFRFUl9NRVRBREFUQSUnCk9SREVSIEJZIGNoZWNrX3N0YXJ0X3RpbWUgREVTQwoK) (the error exists for months already)

Example from [here](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=83459&sha=latest&name_0=PR&name_1=Integration+tests+%28asan%2C+old+analyzer%2C+2%2F6%29):

```
E               Exception: Command [docker compose --project-name roottestmysqlprotocol-gw2 --env-file /ClickHouse/tests/integration/test_mysql_protocol/_instances-0-gw2/.env -f /ClickHouse/tests/integration/helpers/../../../tests/integration/compose/docker_compose_mysql_java_client.yml up --force-recreate -d --no-build] return non-zero code 1:  java1 Pulling 
E                e7c96db7181b Pulling fs layer 
E                f910a506b6cb Pulling fs layer 
E                c2274a1a0e27 Pulling fs layer 
E                91140d9255aa Pulling fs layer 
E                286b3667dcf3 Pulling fs layer 
E                650adee43796 Pulling fs layer 
E                02cb49fb10bc Pulling fs layer 
E                0fdaf3562e89 Pulling fs layer 
E                91140d9255aa Waiting 
E                650adee43796 Waiting 
E                286b3667dcf3 Waiting 
E                02cb49fb10bc Waiting 
E                0fdaf3562e89 Waiting 
E                f910a506b6cb Downloading [==================================================>]     238B/238B
E                f910a506b6cb Download complete 
E                c2274a1a0e27 Downloading [>                                                  ]  538.2kB/70.73MB
E                e7c96db7181b Downloading [>                                                  ]  33.36kB/2.757MB
E                e7c96db7181b Verifying Checksum 
E                e7c96db7181b Download complete 
E                e7c96db7181b Extracting [>                                                  ]  32.77kB/2.757MB
E                e7c96db7181b Extracting [==================================================>]  2.757MB/2.757MB
E                c2274a1a0e27 Downloading [=========>                                         ]  13.88MB/70.73MB
E                e7c96db7181b Pull complete 
E                f910a506b6cb Extracting [==================================================>]     238B/238B
E                f910a506b6cb Extracting [==================================================>]     238B/238B
E                f910a506b6cb Pull complete 
E                c2274a1a0e27 Downloading [================>                                  ]     24MB/70.73MB
E                286b3667dcf3 Downloading [>                                                  ]  33.33kB/2.347MB
E                286b3667dcf3 Verifying Checksum 
E                286b3667dcf3 Download complete 
E                91140d9255aa Downloading [=>                                                 ]  16.38kB/565.3kB
E                91140d9255aa Downloading [==================================================>]  565.3kB/565.3kB
E                91140d9255aa Verifying Checksum 
E                91140d9255aa Download complete 
E                c2274a1a0e27 Downloading [=======================>                           ]  33.58MB/70.73MB
E                650adee43796 Downloading [==================================================>]      94B/94B
E                650adee43796 Verifying Checksum 
E                650adee43796 Download complete 
E                c2274a1a0e27 Downloading [===============================>                   ]  44.28MB/70.73MB
E                02cb49fb10bc Downloading [==================================================>]  1.739kB/1.739kB
E                02cb49fb10bc Verifying Checksum 
E                02cb49fb10bc Download complete 
E                c2274a1a0e27 Downloading [======================================>            ]   54.4MB/70.73MB
E                0fdaf3562e89 Download complete 
E                c2274a1a0e27 Downloading [=============================================>     ]  64.55MB/70.73MB
E                c2274a1a0e27 Verifying Checksum 
E                c2274a1a0e27 Download complete 
E                c2274a1a0e27 Extracting [>                                                  ]  557.1kB/70.73MB
E                c2274a1a0e27 Extracting [====>                                              ]  6.128MB/70.73MB
E                c2274a1a0e27 Extracting [==============>                                    ]  20.05MB/70.73MB
E                c2274a1a0e27 Extracting [==========================>                        ]  37.88MB/70.73MB
E                c2274a1a0e27 Extracting [=====================================>             ]  52.92MB/70.73MB
E                c2274a1a0e27 Extracting [==============================================>    ]  65.18MB/70.73MB
E                c2274a1a0e27 Extracting [==================================================>]  70.73MB/70.73MB
E                c2274a1a0e27 Extracting [==================================================>]  70.73MB/70.73MB
E                c2274a1a0e27 Pull complete 
E                91140d9255aa Extracting [==>                                                ]  32.77kB/565.3kB
E                91140d9255aa Extracting [==================================================>]  565.3kB/565.3kB
E                91140d9255aa Extracting [==================================================>]  565.3kB/565.3kB
E                91140d9255aa Pull complete 
E                286b3667dcf3 Extracting [>                                                  ]  32.77kB/2.347MB
E                286b3667dcf3 Extracting [==================================================>]  2.347MB/2.347MB
E                286b3667dcf3 Pull complete 
E                650adee43796 Extracting [==================================================>]      94B/94B
E                650adee43796 Extracting [==================================================>]      94B/94B
E                650adee43796 Pull complete 
E                02cb49fb10bc Extracting [==================================================>]  1.739kB/1.739kB
E                02cb49fb10bc Extracting [==================================================>]  1.739kB/1.739kB
E                02cb49fb10bc Pull complete 
E                0fdaf3562e89 Extracting [==================================================>]  3.852kB/3.852kB
E                0fdaf3562e89 Extracting [==================================================>]  3.852kB/3.852kB
E                0fdaf3562e89 Pull complete 
E                java1 Pulled 
E               Error response from daemon: No such image: clickhouse/mysql-java-client:b4115cf4c3aca56554c7_amd
```

--- a/tests/ci/integration_tests_runner.py
+++ b/tests/ci/integration_tests_runner.py
@@ -231,7 +231,7 @@ class ClickhouseIntegrationTestsRunner:
         cmd = (
             f"cd {self.repo_path}/tests/integration && "
             f"timeout --verbose --signal=KILL 1h ./runner {self._get_runner_opts()} {image_cmd} "
-            "--command ' echo Pre Pull finished ' "
+            "--pre-pull --command ' echo Pre Pull finished ' "
         )
 
         for i in range(5):

--- a/tests/integration/runner
+++ b/tests/integration/runner
@@ -483,7 +483,7 @@ if __name__ == "__main__":
     cmd = cmd_base + " " + args.command
     cmd_pre_pull = (
         f"{cmd_base} find /ClickHouse/tests/integration/compose -name docker_compose_*.yml "
-        r"-exec docker compose -f '{}' pull \;"
+        r"-exec docker compose -f '{}' pull --quiet \;"
     )
 
     containers = subprocess.check_output(

